packages/ember-glimmer/lib/environment.js

```diff
@@ -31,13 +31,49 @@ import { default as log } from './helpers/log';
 import { default as readonly } from './helpers/readonly';
 import { default as unbound } from './helpers/unbound';
 import { default as classHelper } from './helpers/-class';
+import { default as inputTypeHelper } from './helpers/-input-type';
 import { default as queryParams } from './helpers/query-param';
 import { OWNER } from 'container/owner';
 
 const builtInComponents = {
   textarea: '-text-area'
 };
 
+function buildTextFieldSyntax({ args, templates }, getDefinition) {
+  let definition = getDefinition('-text-field');
+  wrapClassBindingAttribute(args);
+  wrapClassAttribute(args);
+  return new CurlyComponentSyntax({ args, definition, templates });
```

Comment:
This combo is in this file at least three times:

```js
wrapClassBindingAttribute(args);
wrapClassAttribute(args);
return new CurlyComponentSyntax({ args, definition, templates });
```

Is this something we could extract and share?